### PR TITLE
Redundant return

### PR DESCRIPTION
**Pull request by shleeable**
Not exactly world ending... but you know

### CODE REVIEW

**Review by mjankowski — APPROVED** [review]

**Comment by shleeable** [comment]
@renchap this is the worlds least important PR but it's an easy merge.

File: app/controllers/api/web/embeds_controller.rb [hunk 1]
```diff
@@ -20,7 +20,7 @@ def show
       begin
         oembed[:html] = Sanitize.fragment(oembed[:html], Sanitize::Config::MASTODON_OEMBED)
       rescue ArgumentError
-        return not_found
+        not_found
```
Comment (renchap) [inline app/controllers/api/web/embeds_controller.rb:23]: I think the `return` is important here. With your code, what will be executed if there is an `ArgumentError` is:

```ruby
not_found

render json: oembed
```

The `render json: oembed` needs to be in the `begin` block if you do want the same behaviour

**Review by renchap — CHANGES_REQUESTED** [review]

**Comment by mjankowski** [comment]
With the linked PR merged, if you rebase this you'll see a failure with just this change.

**Comment by shleeable** [comment]
ooh in that case. Don't want to cause a regression.

**Pull request closed without merging**